hera_sim/sigchain.py

```diff
@@ -39,7 +38,7 @@ def gen_bandpass(fqs, ants, gain_spread=0.1):
     See Also:
         :meth:`~gen_gains`: uses this function to generate full gains.
     """
-    bp_base = np.polyval(HERA_NRAO_BANDPASS, fqs)
+    bp_base = np.polyval(_get_hera_bandpass(), fqs)
```

Comment:
this should be a parameter to the function, and again, set its default to `None`.

I really don't think you should even need to define a `_get_hera_bandpass` function: the `@_defaults` should have the relevant logic to apply this automatically, if it finds such an argument in the signature.